site/docs/v1/tech/apis/users.adoc

```diff
@@ -520,6 +521,41 @@ include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
 :never_search_error!:
 :never_missing!:
 
+== Password Hashes
```

Comment:
Should this section come after the Import User section instead of the Import Refresh Token section?

Reply:
I'm not too attached either way. In my mind, all three of these are parts of 'importing user data', but happy to move it up if you think that is important.

Reply:
Will move it up, no problemo.